bucket/filezilla.json

```diff
@@ -15,10 +15,15 @@
     },
     "extract_dir": "FileZilla-3.47.1",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
-        "}"
+        "@\"",
```

Comment:
You do not need double quotes when no variable is used

Reply:
it depend if i need to add the fullpath from the previous comment. let me know.

Reply:
since i added the fullpath, the $dir variable is now used